react-function-components/src/index.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+function CustomButton() {
+  return <button>Click Me!</button>;
+}
+
+ReactDOM.render(CustomButton(), document.getElementById('root'));
```

Comment:
In react we never really call our own components, instead, we create react elements with them.

```jsx
ReactDOM.render(
  <CustomButton />,
  document.querySelector('#root')
);
```